udata_piwik/metrics.py

```diff
@@ -167,7 +189,8 @@ def get_value(self):
 def aggregate_datasets_daily(org, day):
     keys = ['datasets_{0}'.format(k) for k in KEYS]
     ids = [d.id for d in Dataset.objects(organization=org).only('id')]
-    metrics = Metrics.objects(object_id__in=ids, level='daily', date=day.isoformat())
+    date = day.isoformat()
```

Comment:
`date` is a reserved keyword and it introduces a useless temp var, I would rather add a linebreak before `object_id__in`.